src/testing.sh

```diff
@@ -1457,20 +1481,55 @@ rlIsOSLike() {
   return 0
 }
 
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# rlIsOSVersion
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+: <<=cut
+=pod
+
+=head3 rlIsOSVersion
+
+    rlIsOsVersion [Num|opNum]
+
+=over
+
+=item Num|opNum
+
+Argument is based on VERSION_ID in /etc/os-release.
+Range matching can be used in the form used by C<rlIsRHEL>.
+
+=back
+
+Returns 0 when we're running distribution of particular version
+requested by parameter.
+It usually follows after C<rlIsOS> and C<rlIsOSLike>.
+
+    rlIsOSVersion 9 10
+
+Returns 0 if we are running distribution with VERSION_ID 9 or 10.
+
+=cut
+#'
 
 rlIsOSVersion() {
   [[ -z "$1" ]] && return
```

Comment:
```suggestion
  [[ -z "$1" ]] && {
    rlLogDebug "$FUNCNAME(): no version specified, acting as noop"
    return 0
  }
```
it would be nice to put that `$FUNCMANE(): ` to all of the debug logs.